Additional docs (#171)
ArbitrumFoundation/governance: docs/security-council-nominee-vetting.md

# Nominee vetting guidelines

> 7 days after an election has begun there is a 14 day vetting period to allow the Foundation to conduct compliance checks on each of the nominees that have received 0.2% of votable tokens. They can then exclude nominees who fail these checks, and include nominees if not enough nominees are available. The Member Election Governor authorises a specific address that is controlled by the Foundation to do these actions.

Reviewer: ```suggestion
7 days after an election has begun there is a 14 day vetting period to allow the Foundation to conduct compliance checks on each of the nominees that have received 0.2% of votable tokens. They can then exclude nominees who fail these checks, and include nominees if fewer than 6 nominees have been selected in the Nominee Election phase. The Member Election Governor authorises a specific address that is controlled by the Foundation to do these actions.
```

- Is the owner of the address able to create a signature of all chains where a Security Council is located that will be accepted by a Gnosis Safe?

If any of the above are false the Foundation should call `excludeAddress` to stop that address from proceeding to the next stage.

### Including addresses

During the 14 day vetting period the Foundation should include new addresses if the number of nominees that achieved a threshold of 0.2% of votable tokens and have not been excluded is less than 6. They should include new addresses until there are a total of 6 nominees that will progress to the next stage. New addresses should be selected in the following order, if each stage does not yield enough included addresses to make up to 6 nominees then the Foundation should progress to the next stage:

1. Members from the outgoing cohort that fulfil the requirements to not be excluded should be chosen at random.
2. Contenders who did not receive 0.2% threshold of votes should be included, in order of votes descending

> 3. If there still aren’t 6 nominees the Foundation may include any address they wish

Reviewer: ```suggestion
3. If there still aren’t 6 nominees the Foundation may include any address they see fit```

> 3. If there still aren’t 6 nominees the Foundation may include any address they see fit```

Reviewer: ```suggestion
3. If there still aren’t 6 nominees the Foundation may include any address they see fit
```
